dist/neon.js

```diff
@@ -792,16 +804,34 @@ neon.query.Connection.ELASTICSEARCH = 'elasticsearch';
  * address:port pair (e.g., localhost:9300).  If no port is provided, the Neon server will assume the default port
  * for the databaseType:  27017 for Mongo, 10000 for Spark via a Hive2 Thrift connection, and 9300 for an
  * Elasticsearch transport client.
+ * @param {Boolean} listenForUpdates [true] Whether or not the connection should automatically listen for updates on connection
  */
-neon.query.Connection.prototype.connect = function(databaseType, host) {
+neon.query.Connection.prototype.connect = function(databaseType, host, listenForUpdates) {
     this.host_ = host;
     this.databaseType_ = databaseType;
     this.messenger.publish(neon.eventing.channels.CONNECT_TO_HOST, {
         host: host,
         type: databaseType
     });
+    if (listenForUpdates !== false) {
```

Comment:
Why `!== false`?  Shouldn't we just use `if (!listenForUpdates)` since that would check for all falsey values (like undefined)?

Reply:
Undefined here is being treated as true, basically unless you explicitly set it to false, it should listen for updates

Reply:
Yeah, but is that what we want?  Should the default behavior avoid making the extra connection to the server?

If we want the default to be listen-for-updates, then maybe the argument should be the opposite (like "ignoreUpdates") so that undefined values and false values behave the same way.